Jenkinsfile

```diff
@@ -106,22 +106,18 @@ pipeline {
                 }
             }
         }
-        stage('Sanity Test') {
+        stage('VRT / Ally Tests') {
             steps {
-                script { env.BUILD_STAGE = 'Sanity Test' }
+                script { env.BUILD_STAGE = 'VRT / Ally Tests' }
```

Comment:
Use A11y here or the full Accessibility, rather than Ally.

Reply:
Should they be grouped together. Or split out into their own stages? Whilst I'm here I can tackle it.

Reply:
Granular steps makes sense to me. Don't need to be grouped together